libcxx/include/__algorithm/set_intersection.h

```diff
@@ -95,6 +98,14 @@ __set_intersection(
     _Compare&& __comp,
     std::forward_iterator_tag,
     std::forward_iterator_tag) {
+#if _LIBCPP_HARDENING_MODE == _LIBCPP_HARDENING_MODE_DEBUG
+  if (!__libcpp_is_constant_evaluated()) {
```

Comment:
Why is this required?

Reply:
I still don't understand. Can you try removing the `if(is-constant-evaluated)` and let's see what the CI says? You're probably right, but I'd like to see what the error is.

Reply:
+1 -- also curious to see the error.

Reply:
Yep, you're both right, that was bogus! Thanks for challenging me on it :).